synapse/cortex.py

```diff
@@ -5277,8 +5291,10 @@ async def getNodeByNdef(self, ndef, view=None):
 
         buid = s_common.buid((form.name, norm))
 
-        async with await self.snap(view=view) as snap:
-            return await snap.getNodeByBuid(buid)
+        if view is None:
```

Comment:
i wonder if we can remove this api too. it seems like its only used for test purposes?

Reply:
Yeah I don't think we need it just for test purposes, I'll remove it.